tests/cases/fourslash/getJavaScriptQuickInfo7.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path="fourslash.ts" />
+
+// @allowNonTsExtensions: true
+// @Filename: file.js
+//// /**
+////  * This is a very cool function that is very nice.
+////  * @returns something
+////  */
+//// function a1() {
+//// 	try {
+//// 		throw new Error('x');
+//// 	} catch (x) { x--; }
+//// 	return 23;
+//// }
+//// 
+//// x - /**/a1()
+
+goTo.marker();
+verify.quickInfoExists();
```

Comment:
Can we also verify that the quickinfo still correctly contains the first line of the comment?